tests/models/test_tpu.py

```diff
@@ -124,18 +128,19 @@ def test_model_16bit_tpu_cores_1(tmpdir):
     tpipes.run_model_test(trainer_options, model, on_gpu=False)
 
 
-@pytest.mark.parametrize("tpu_core", [1, 5])
+@pytest.mark.parametrize("devices", [1, 5])
```

Comment:
Let's keep this as tpu_core as well. We can address with deprecation warning in a separate PR

Reply:
The assert is to test tpu_cores pass as a list, we want to keep a test for that
And these two tests failed pre-commit because L88 and L 150 couldn’t find  tpu_cores

Reply:
Ok